lib/event_sourcery/postgres/config.rb

```diff
@@ -26,14 +26,23 @@ def initialize
         @auto_create_projector_tracker = true
       end
 
+      # The event store.
+      #
+      # @return [EventStore]
       def event_store
         @event_store ||= EventStore.new(event_store_database)
       end
 
+      # The event source.
+      #
+      # @return [EventSource]
       def event_source
         @event_source ||= ::EventSourcery::EventStore::EventSource.new(event_store)
       end
 
+      # The event sink.
+      #
+      # @return [EventSink]
```

Comment:
Comments like these add no value. No value is provided over just reading the method name. I think we should remove them.